### PR TITLE
Added more entities

### PR DESCRIPTION
**Pull request by rockeynebhwani**

### CODE REVIEW

**Comment by patrickhulce** [comment]
You'll need to rebase against master @rockeynebhwani  :)

**Comment by rockeynebhwani** [comment]
@patrickhulce  - can you have check now?

File: data/entities.json5 [hunk 1]
```diff
@@ -4212,7 +4212,8 @@
   {
     "name": "Evergage",
     "categories": ["analytics"],
-    "domains": ["*.evergage.com"]
+    "domains": ["*.evergage.com", "*.evgnet.com"]
+	"examples": ["cdn.evgnet.com"]
```
Comment (patrickhulce) [inline data/entities.json5]:
```suggestion
    "examples": ["cdn.evgnet.com"]
```

File: data/entities.json5 [hunk 2]
```diff
@@ -4212,7 +4212,8 @@
   {
     "name": "Evergage",
     "categories": ["analytics"],
-    "domains": ["*.evergage.com"]
+    "domains": ["*.evergage.com", "*.evgnet.com"]
```
Comment (patrickhulce) [inline data/entities.json5]:
```suggestion
    "domains": ["*.evergage.com", "*.evgnet.com"],
```

**Comment by rockeynebhwani** [comment]
Thanks @patrickhulce ... Sorry.. I got distracted with something else.. will take care in future

**Pull request merged**